Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -5,6 +5,9 @@
    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>  
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="ClientApp\util\permissions.ts" />
```

Comment:
@jcubbage hmm, this shouldn't be here, and actually i don't think there should be any csproj changes needed.  I'm especially worried about the typescript compile item group farther down in this file -- none of the other *.ts files need this, webpack is going to do the build